File: Logistics.Web/ClientApp/src/app/models/funding-account.ts
import { FundTransaction } from './fund-transaction';
import { FundUser } from './fund-user';
import { ItemGroup } from './item-group';

export class FundingAccount {
  id: number;
  name: string;
  fundTransactions: FundTransaction[];
  fundUsers: FundUser[];
  itemGroups: ItemGroup[];
}

File: Logistics.Data/FundingAccount.cs
using System.Collections.Generic;

namespace Logistics.Data
{
    public class FundingAccount
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public List<FundTransaction> FundTransactions { get; set; }
        public List<FundUser> FundUsers { get; set; }
        public List<ItemGroup> ItemGroups { get; set; }
    }
}
